### PR TITLE
Updates to the step-63 introduction.

### PR DESCRIPTION
**Pull request by bangerth**
These mostly add background information to users not yet familiar with
the material.

### CODE REVIEW

**Comment by bangerth** [comment]
@tjhei , @tcclevenger : FYI.

File: examples/step-63/doc/intro.dox [hunk 1]
```diff
@@ -75,31 +109,54 @@ for each cell, and $\delta_K$ is a cell-wise constant
...
+<h3>Smoothers</h3>
 
 One of the goals of this tutorial is to expand from using a simple
 (point-wise) Gauss-Seidel (SOR) smoother that is used in step-16
 (class PreconditionSOR) on each level of the multigrid hierarchy.
-Here, we consider point-wise smoothers (Jacobi and SOR) and cell-based
-smoothers (Block Jacobi and Block SOR). The cell-based smoothers can
+The term "point-wise" is traditionally used to indicate that one
+solves at one "grid point" at a time; for scalar problems, this means
+to use a solver or smoother that solves for one unknown of the linear
```
Comment (tjhei) [inline examples/step-63/doc/intro.dox]:
```suggestion
to use a solver or smoother that updates one unknown of the linear
```

File: examples/step-63/doc/intro.dox [hunk 2]
```diff
@@ -75,31 +109,54 @@ for each cell, and $\delta_K$ is a cell-wise constant
...
+system at a time, keeping all of the others fixed; one would then
+iterate over all unknowns in the problem and, once done, start over again
+from the first unknown until these "sweeps" converge. Jacobi,
```
Comment (tjhei) [inline examples/step-63/doc/intro.dox]: I find this confusing as it seem to imply you have to repeat until "convergence", which is not true for a smoother.

File: examples/step-63/doc/intro.dox [hunk 3]
```diff
@@ -75,31 +109,54 @@ for each cell, and $\delta_K$ is a cell-wise constant
...
+Gauss-Seidel, and SOR iterations can all be interpreted in this way.
+
+But these methods are known to converge rather slowly when used as
+solvers, and to act as smoothers of only mediocre quality when used in
```
Comment (tjhei) [inline examples/step-63/doc/intro.dox]: I wouldn't call point smoothers "mediocre". They are very performant and, in fact, all our large scale parallel computations use point smoothers. In the results section, you can see that they easily beat the cell based smoothers in runtime.

**Comment by bangerth** [comment]
Good points. I've addressed these all.

**Pull request merged**